src/main/java/com/synopsys/integration/alert/provider/blackduck/validators/BlackDuckValidator.java

```diff
@@ -100,6 +101,13 @@ public boolean validate(BlackDuckProperties blackDuckProperties) {
                     valid = false;
                 }
             }
+
+            BlackDuckApiTokenValidator blackDuckAPITokenValidator = new BlackDuckApiTokenValidator(blackDuckProperties);
+            if (!blackDuckAPITokenValidator.isApiTokenValid()) {
+                String message = "User permission failed, cannot read notifications from Black Duck.";
```

Comment:
We should probably either just pass this String directly into the method, or create a constant for it. Otherwise I believe SonarCloud will flag it as a code smell.